app/controllers/experiments_controller.rb

```diff
@@ -630,4 +633,22 @@ def sort_my_modules(records, sort)
       records
     end
   end
+
+  def generate_two_by_two_connections(src_tasks, dest_tasks)
```

Comment:
should maybe be more descriptive, maybe something like `reconnect_my_modules_after_archive`